sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -233,8 +233,8 @@ def _iter_contextual_wrapper(self, max_wait_time=None):
             # different max_wait_times to different iterators and uses them in concert.
             if max_wait_time:
                 # _timeout to _idle_timeout
-                original_timeout = self._handler._idle_timeout
```

Comment:
I had made this idle_timeout earlier on, that was incorrect. this timeout is specifically for the iterator...not related to the connection timing out